source/Features.rst

```diff
@@ -527,7 +542,7 @@ EC2 Product
 ----------- 
 
 Researchers can login to the portal and quickly order  EC2 products.
-Find the Provisioned EC2 product i.e. Ayush Medicine in the Active Products panel. Or click on the “View All” button to get a list of all provisioned products.
+Find the Provisioned EC2 product i.e. EnvironmentalProtectionAgency in the My Products panel. Or click on the “View All” button to get a list of all provisioned products.
```

Comment:
Why are we mentioning specific products which are in our environment. Is it because they are displayed in the screen-shot?

Reply:
Yes.